include/ChaiVM/utils/file-format/klass-info.hpp

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "vector"
+#include <cstdint>
+
+#include "ChaiVM/interpreter/instruction.hpp"
+#include "ChaiVM/types.hpp"
+#include "ChaiVM/utils/io-bytes.hpp"
+
+namespace chai::utils::fileformat {
+
+enum FieldTag : uint8_t { I64 = 1, F64 = 2 };
+
+struct FieldInfo {
+    interpreter::Immidiate name_;
+    uint8_t isObject_;
```

Comment:
@levBagryansky why can't you make this field of type `FieldTag`?

Reply:
@c71n93 `0` means that it is primitive type.  `FieldTag` is in the field `tagOrKlassNum_`.

Reply:
@levBagryansky So, as I understood, klass num can't be 1 or 2?

Reply:
No, it can. If `isObject` is 1 that `tagOrKlassNum_` is klass number (0, ..). Otherwise `tagOrKlassNum_` this is int or float (1 or 2)